README.rst

```diff
@@ -166,6 +166,17 @@ but in some cases (such as running ``tox --current-env``, uninstalling the
 plugin, and running ``tox``), you will get undefined results
 (such as installing packages from PyPI into your current environment).
 
+Environment variables are not passed by default
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Although the plug name suggest that current environment is used for tests,
+it means the Python environment, not Shell.
+If you want the tests to see all environment variables of the calling Shell,
```

Comment:
- You don't need to pass all variables
- You don't necessarily need to run tox from a shell
```suggestion
If you want the tests to see environment variables of the calling process,
```